### ISSUE TITLE
Multi-monitor: `take` window  (`Super+T`) and then move mouse to another monitor sometimes causes PaperWM to lose control of other monitor

### ISSUE THREAD

**Issue by jtaala** [paperwm/PaperWM#631]
**To Reproduce**
1. use multi-monitors
2. `take` window
3. while `take` window, move mouse to the other monitor
4. See inconsistent PaperWM state

~~Happens on both gnome 44 and gnome 45.~~
Happens on gnome 44 (and likely 43, 42).  This particular issue is fixed by #632 on gnome 45.

```
Distribution: Fedora Linux
GNOME Shell 44.5
Display server: Wayland
PaperWM branch/tag: gnome-44
PaperWM commit: 4faa751a1be54987e253e8569ef6f209016b4131
Enabled extensions:
- paperwm@paperwm.github.com
```

@Lythenas, is related to `ClickOverlay` and looks similar to the end state mentioned in #627.

**Comment by jtaala** [comment]
Reopening this issue for Gnome 44 (where it occurs sometimes).

**Comment by jtaala** [comment]
Closed by #633
